lib/tasks/data_demo.rake

```diff
@@ -5,6 +5,35 @@ namespace :data do
 
     conference = create(:full_conference, title: 'Open Source Event Manager Demo', short_title: 'osemdemo' ,description: "This is a [Open Source Event Manager](http://osem.io/) demo instance. You can log in as **admin** with the password **password123** or just you just [sign up](/accounts/sign_up) with your own user. We hope you enjoy checking out all the functionality, if you have questions don't hesitate to [contact us](http://osem.io/#contact)!\r\n\r\n## Data will be destroyed every thirty minutes or whenever someone updates the [OSEM source code on github](https://github.com/openSUSE/osem/commits/master).")
     conference.contact.update_attributes(email: 'osemdemo@osem.io', sponsor_email: 'osemdemo@osem.io')
-    create(:admin, email: 'admin@osem.io', username: 'admin', password: 'password123', password_confirmation: 'password123')
+    create(:conference, title: 'Demo Conference', short_title: 'Opensuse' ,description: "This is a demo conference instance. No information about this conference is set by default.")
```

Comment:
The coma before `description` should have the space after and not before:
`create(:conference, title: 'Demo Conference', short_title: 'Opensuse', description: "This is a demo conference instance. No information about this conference is set by default.")`
:wink: